nautobot/docs/release-notes/version-2.3.md

```diff
@@ -12,6 +12,10 @@ This document describes all new features and changes in Nautobot 2.3.
 
 Added an optional `role` field to Interface and VMInterface models to track common interface configurations. Now the users can create [Role](../user-guide/platform-functionality/role.md) instances that can be assigned to [interfaces](../user-guide/core-data-model/dcim/interface.md) and [vminterfaces](../user-guide/core-data-model/virtualization/vminterface.md).
 
+#### Saved Views
+
+Added the new model `SavedView`, which gives users the ability to save multiple different list view configurations (table configuration, filter parameters, pagination and sorting) into Saved View instances for ease of access.
```

Comment:
This is a very data-model-focused way to describe the feature. Can we make it more user focused - "Users now have the ability to save their configuration of list views (table columns, filtering, pagination and sorting) for ease of later use and reuse", something like that?